### PR TITLE
Update logs from `system76-driver` to match support panel

### PR DESCRIPTION
**Pull request by cwsystem76**
This commit updates the logs generated by `system76-driver` so they match the ones generated by the Support panel after PR https://github.com/pop-os/support-panel/pull/12.

Specifically, it updates the `dump_logs` function in `util.py` so the output filenames and commands match the Support panel output.

### CODE REVIEW

**Review by mmstick — APPROVED** [review]

**Review by leviport — CHANGES_REQUESTED** [review]
Looks like this is working as expected, but PRs to system76-driver should be done according to the instructions in the readme.

**Comment by cwsystem76** [comment]
> Looks like this is working as expected, but PRs to system76-driver should be done according to the instructions in the readme.

This didn't seem like it was worth a version bump but I'll go ahead and do so.

**Comment by cwsystem76** [comment]
@leviport  Running the version bump script results in the following:
```
Will bump 'focal' version from '20.04.52' to '20.04.53~~alpha'
```
I said "no" when it asked to confirm because this doesn't look right to me. Is there a script

**Comment by leviport** [comment]
That's correct, see https://github.com/pop-os/system76-driver/pull/237/files for reference. Running `make-release.py` afterwards removes the `~~alpha` and the `Daily WIP` line.

**Comment by cwsystem76** [comment]
@leviport  @mmstick Done. The script bumped the version and automatically pushed the changes, so the existing reviews will need to be redone. Sorry about that.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.53~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.53
```
Comment (leviport) [inline debian/changelog:3]: Don't forget the changelog entry under the Daily WIP line

Comment (cwsystem76) [reply debian/changelog:3]: Done.

**Review by leviport — APPROVED** [review]
I rebased this so the version bump commit comes first, and I squashed the changelog commit into the main commit. Should be good to go now.

**Review by mmstick — APPROVED** [review]

**Comment by mmstick** [comment]
Ready to merge then?

**Comment by leviport** [comment]
Yep!

**Comment by cwsystem76** [comment]
Thanks, folks.

**Pull request merged**